about.html

```diff
@@ -0,0 +1,133 @@
+      </div>
+        <p class="about__date--time">Please contact us on this Email for more information on the GH Comedy Crusade 2023</p>
+        <a href="#" class="about__date--email">ghcomedy@gmail.com</a>
+      </div>
+    </header>
+  </div>
+  <!-- End of Header -->
+
+  <!-- Main Content -->
+  <main class="main">
+    <section class="container">
+      <div class="about__main">
+        <h2 class="about__main--title">GH Comedy Crusade 2023 Logo</h2>
+        <p class="about__main--description">This is the official logo for the first ever GH Comedy Crusade</p>
+      </div>
+
+      <div class="about__logo">
+       <h2 class="about__logo--title"><img src="./images/logo.jpg" alt="GH Comedy Crusade Logo"></h2>
+      </div>
+    </section>
+    <section class="event">
+      <div class="container">
+        <h2 class="event__main--title">See Past Events</h2>
+        <p class="about__main--description">These are memories created by some past events. In case you missed it, you have the change to be part of this years events.</p>
+        <div class="event__content">
+          <div class="event__content--img">
+            <img src="./images/pastevent-2.jpg" alt="Comedian Khemikal Performing on stage">
+            <div class="event__content--text">
+              <h2>2021</h2>
+              <p>Absent Minded at Airport Hill</p>
+            </div>
+          </div>
+          <div class="event__content--img">
+            <img src="./images/too-cute.jpg" alt="Group Picture of Comedians">
+            <div class="event__content--text">
+              <h2>2022</h2>
+              <p>Too Cute To Be Mute at National Theatre</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Partner -->
+    <section class="partner">
+      <div class="container">
+        <h2 class="partner__title">Sponsors</h2>
+        <div class="partner__content">
+        <img src="./images/comedycrusade.png" alt="Comedy Crusade Logo" class="partner__logo">
+        <img src="./images/tv3.png" class="partner__logo" alt="TV3 Logo" class="partner__logo">
+        <img src="./images/GCB.png" alt="GCB Logo" class="partner__logo">
+        <img src="./images/fanmilk.png" alt="Fan Milk Logo" class="partner__logo">
+        <img src="./images/goil.png" alt="Goil Logo" class="partner__logo">
+        </div>
+      </div>
+    </section>
+    <!-- End of Partner -->
+  </main>
+  <!-- End of Main Content -->
+
+  <!-- Footer -->
+  <footer class="footer">
+    <div class="container">
+      <div class="footer__content">
+        <div class="footer__logo">
+          <img src="./images/logo.png" alt="">
```

Comment:
- [ ] While most of your images have an `alt` attribute, kindly don't forget to add it to the rest :+1: to make your page even more accessible.